.github/arc-runner-config.yaml

```diff
@@ -15,6 +15,14 @@ runnerConfig:
     runnerLabel: linux.2xlarge
     containerMode: dind-rootless
     workingDiskSpace: 150Gi
+  - cpu: 8.0
+    maxRunners: 300
+    memory: 16128Mi
+    minRunners: 0
+    nodeType: compute-amd64-avx512
+    runnerLabel: linux-avx512.2xlarge
```

Comment:
maybe, lets keep the standard of `os`.`instance-size`.`add-capability-1`.`add-capability-2`.`add-capability-n`

so maybe it could be better to call it `linux.2xlarge.avx512`?

Reply:
(in this case we might want to update https://github.com/pytorch/test-infra/pull/5079/files#diff-44d53c0cf3072ca763edab85934b5ef2e369745676eaebdea148f04f7eb8913bL24 as well)

Reply:
Updated per the suggestion.